lab-max/app/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html ng-app="cowsayApp">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title> Cowsay-Angular Lab </title>
+  </head>
+  <body>
+    <main ng-controller="CowController as cowCtrl" ng-init="cowCtrl.activeCow = 'default'">
+      <section>
+        <h1> {{cowCtrl.title}} </h1>
+        <h3> {{cowCtrl.intro}} </h3>
+        <form
+        novalidate
```

Comment:
@meeksfred best to indent these over 1 tab (2 spaces) if you are breaking them on to new lines